ska_trend/fid_drop_mon/update.py

```diff
@@ -96,7 +100,11 @@ def get_vehicle_only_intervals(
     stop = CxoTime(stop) if stop is not None else CxoTime.now()
     start = sosa_patch if start is None else np.max([CxoTime(start), sosa_patch])
```

Comment:
You don't need to fix this here, but in general you should use `max(a, b)` for the max of exactly two objects. 

`np.max` is intended for the max of a numpy array, with args that relate to arrays. When passed a list, `np.max` has to convert that to a numpy array first.